src/i18n/resources/zh/dataSource.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   referenceToTataSource: '数据源参考文献',
   typeOfDataSource: '数据源的类型',
+  typeOfDataSourceText: '$t(dataSource.typeOfDataSource)',
```

Comment:
what about fr and ru ?

Reply:
We are still pending translations for the whole section for these languages, so we are also missing the initial key.
![image](https://user-images.githubusercontent.com/5508251/227155967-3d1c1ca1-413d-418a-a852-ccaea6832309.png)

I will add issue about this so I remember to add when we get the translations for previous